packages/components/src/card/test/index.tsx

```diff
@@ -56,9 +56,9 @@ describe( 'Card', () => {
 			const { container: containerSquared } = render(
 				<Card isRounded={ false }>Code is Poetry</Card>
 			);
-			expect( containerRounded.firstChild ).toMatchStyleDiffSnapshot(
-				containerSquared.firstChild
-			);
+			expect(
+				containerRounded.firstElementChild
+			).toMatchStyleDiffSnapshot( containerSquared.firstElementChild );
```

Comment:
We also need to rename `packages/components/src/card/test/__snapshots__/index.js.snap` to `packages/components/src/card/test/__snapshots__/index.tsx.snap`